website/docs/api/regions.mdx

```diff
@@ -39,8 +39,8 @@ h3.polyfill(polygons, res, geo_json_conformant=False)
 <TabItem value="java">
 
 ```java
-List<Long> polyfill(List<GeoCoord> points, List<List<GeoCoord>> holes, int res);
-List<String> polyfillAddress(List<GeoCoord> points, List<List<GeoCoord>> holes, int res);
+List<Long> polyfill(List<LatLng> points, List<List<LatLng>> holes, int res);
+List<String> polyfillAddress(List<LatLng> points, List<List<LatLng>> holes, int res);
```

Comment:
These Java ones are anticipating that the binding will change, not sure if that makes sense at this point?

Reply:
We should use consistent terms in the bindings as well, so this makes sense to me.